server/metrics/units.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2022-2023 Tigris Data, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metrics
+
+func getUnitsFromBytes(bytes int64, unitSize int) int64 {
+	if bytes == 0 {
+		return 0
+	}
+	return bytes/int64(unitSize) + 1
```

Comment:
this will count as an extra unit in case bytes are divided by unitSize in equal chunks, better to do this?
```
if bytes % unitSize == 0 {
  return bytes/unitSize
}
return bytes/int64(unitSize) + 1
```

Reply:
Good catch, thanks! Yes, this is better.